src/tests/copy.rs

```diff
@@ -236,6 +236,7 @@ fn copy_multi_test() {
     let child = thread::spawn(move || {
         let mut opts = Options::new();
         opts.foreground(true);
+        opts.offer_additional_text_mimes(true);
```

Comment:
Won't be needed once the current default behavior is preserved.

But I would also like a test for the new behavior.